From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

alexandair committed:
Fixed conflict

rojasja committed:
Merge pull request #2 from MicrosoftDocs/master

Syncing from original

ccompy committed:
Merge pull request #4 from MicrosoftDocs/master

update my master

raiye committed:
Update cloud-services-guestos-update-matrix.md

ccompy committed:
update management addresses

I missed two addresses

raiye committed:
Update cloud-services-guestos-update-matrix.md

raiye committed:
Update cloud-services-guestos-msrc-releases.md

raiye committed:
Update cloud-services-guestos-msrc-releases.md

Commit:
Update stream-analytics-run-a-job.md

Commit:
edit pass: api-management-using-with-internal-vnet

LoriMerrick committed:
Update api-management-using-with-internal-vnet.md

LoriMerrick committed:
Changed the format on one of the footnotes.

rojasja committed:
Merge pull request #3 from MicrosoftDocs/master

Updating my fork

LoriMerrick committed:
Made a small change per the author

LoriMerrick committed:
Merge branch 'master' into api-management-two-articles

LoriMerrick committed:
Merge branch 'master' into api-management-two-articles

tdykstra committed:
importing text as-is

gopitk committed:
Link to missing tools overview page

ggailey777 committed:
in process update for v2

BryanLa committed:
freshness updates

tdykstra committed:
acrolinx, images, links

tdykstra committed:
acrolinx, images, links 2

Commit:
apps and svcs as arm resources

v-vikum2 committed:
USER STORY 1091602 SaaS App Tutorial: TurboRater

ggailey777 committed:
Add missing v2.0 topics

shruja committed:
USER STORY 1091636: SaaS App Tutorial: Schoox

BryanLa committed:
updates

BryanLa committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into bryanla-integrating-apps

BryanLa committed:
freshness

Commit:
fixes

BryanLa committed:
freshness

tdykstra committed:
acrolinx images links 3

ggailey777 committed:
Add app setting steps

BryanLa committed:
freshness

Commit:
edit pass: service-fabric-understand-and-troubleshoot-with-system-health-reports

BryanLa committed:
updates

ggailey777 committed:
Acrolinx fixes

BryanLa committed:
updates

danieldobalian committed:
Updated CA doc for Graph 403 Error

Commit:
further fixes

Commit:
acrolinx fix

shruja committed:
update

rachel-msft committed:
adding how to configure server parameters in cli article

rachel-msft committed:
Update howto-configure-server-parameters-using-cli.md

shruja committed:
USER STORY 1093032: SaaS App Tutorial: Agiloft

shruja committed:
update

Commit:
Changes to pronunciation

various minor edits

Commit:
Updates to our FAQ

Minor updates to our FAQ document

Commit:
changes suggested by raised PR-scorecard

minor edits following scorecard recommendation

rojasja committed:
Adding articles on linked in and twitter

rojasja committed:
Updating TOC with new files

li, rest1,rest2,rest3, and twitt

rojasja committed:
Correcting some broken links

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into service-fabric-understand-and-troubleshoot2

Commit:
edit pass: service-fabric-understand-and-troubleshoot-with-system-health-reports

BryanLa committed:
updates

rojasja committed:
Adding migration and UI files- new

rojasja committed:
deleting unauthorized file

a txt file was deleted

LoriMerrick committed:
Made a small edit.

LoriMerrick committed:
Made additional small edits.

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

tdykstra committed:
acrolinx, images, links

rojasja committed:
Updating TOC

ggailey777 committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into ignite

rojasja committed:
additional edits and fixing TOC

tdykstra committed:
more acrolinx etc. and link fixes

ccompy committed:
Update management-addresses.md

text changes

rloutlaw committed:
add plugins to toc, update CTA

tdykstra committed:
more acrolinx etc.

tdykstra committed:
more acrolinx etc.

BryanLa committed:
updates

rachel-msft committed:
Update howto-configure-server-parameters-using-cli.md

tdykstra committed:
singletons, corrections

fhryo-msft committed:
Replace REST samples with Powershell and CLI

Replace REST samples with Powershell and CLI

fhryo-msft committed:
Add notes for classic storage accounts

Add notes for classic storage accounts

tdykstra committed:
sub-orchestrations

olivertowers committed:
Update the quickstart guide to reflect sample project changes.

Sample project is being modified to support .NET Core and build multiple frameworks.

ggailey777 committed:
Tom review updates

shruja committed:
update

ggailey777 committed:
Add TOC entry

shruja committed:
update

BryanLa committed:
updates

BryanLa committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into bryanla-integrating-apps

BryanLa committed:
updates

LoriMerrick committed:
Mad a small edit.

rachel-msft committed:
Update howto-configure-server-parameters-using-cli.md

rojasja committed:
Multiple edits after human review...

LoriMerrick committed:
Made small formatting changes.

Commit:
Grouped HowTo articles

Commit:
task 444: add borders to graphics

Commit:
task 444: add borders to graphics-pt 2

Commit:
Fix

tdykstra committed:
proofread 1

ccompy committed:
text tweaks

tdykstra committed:
proofread 2

v-stbee committed:
Adding troubleshooting guide

Commit:
task 444: add borders to graphics-pt 3

v-stbee committed:
Adding in the deployment troubleshooting reference in the TOC

tdykstra committed:
standardize titles

hning86 committed:
Update how-to-create-dsvm-hdi.md

amitsriva committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

amitsriva committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

Commit:
task 444: add borders to graphics-pt 4

btardif committed:
updateing screenshots for ip restrictions

tdykstra committed:
remove stub

Commit:
Merge pull request #7 from MicrosoftDocs/master

Merging upstream

tdykstra committed:
remove note with link to http-api

Commit:
Fix link and correct ownership on Storage Explorer topics

raiye committed:
Update cloud-services-guestos-msrc-releases.md

raiye committed:
Update cloud-services-guestos-update-matrix.md

CarlRabeler committed:
Update sql-database-intelligent-insights.md

ShawnJackson committed:
Merge pull request #24697 from LoriMerrick/service-fabric-understand-and-troubleshoot2

edit pass: service-fabric-understand-and-troubleshoot-with-system-hea…

paulathurman committed:
edit pass: backup-azure-arm-restore-vms-2

Lisaco88 committed:
Merge pull request #24580 from gopitk/patch-6

Link to missing tools overview page

tdykstra committed:
remove en-us

JasonWHowell committed:
Update TOC.yml

JasonWHowell committed:
Update TOC.yml

rojasja committed:
File updates

rojasja committed:
deleting unnecessary files

PRMerger15 committed:
Merge pull request #24872 from kraigb/master

Fix link and correct ownership on Storage Explorer topics

rwike77 committed:
typo

Commit:
edit pass: active-directory-licensing-group-problem-resolution-azure-portal

tdykstra committed:
clarify portal vs. VS

sethmanheim committed:
Freshness reviews

Commit:
edit pass: active-directory-licensing-group-problem-resolution-azure-portal

tdykstra committed:
standadize next steps link text

GitHubber17 committed:
Merge pull request #24715 from rachel-msft/ServerParametersCLI

adding how to configure server parameters in cli article

Commit:
fixed blocking issues

amitsriva committed:
Enhancements to issues

tdykstra committed:
cory feedback

Commit:
Merge pull request #24873 from CarlRabeler/patch-344

Update sql-database-intelligent-insights.md

amitsriva committed:
Grammar checks

Commit:
Merge pull request #24879 from sethmanheim/freshy

Freshness reviews

ShawnJackson committed:
Update backup-azure-arm-restore-vms.md

amitsriva committed:
More grammar fixes

PRMerger10 committed:
Merge pull request #24878 from rwike77/freshness

typo

olivertowers committed:
Update meta description.

olivertowers committed:
Another update to the meta info.

rjagiewich committed:
Merge pull request #24789 from fhryo-msft/master

Update samples and notes

Lisaco88 committed:
Merge pull request #24769 from rloutlaw/jenkins-toc-updates2

add new plugins to toc, update CTA

ShawnJackson committed:
Merge pull request #24875 from paulathurman/backup-azure-arm-restore-vms-2

edit pass: backup-azure-arm-restore-vms-2

Commit:
note not formatted correctly

rojasja committed:
Last corrections on format

amitsriva committed:
grammar fix

BryanLa committed:
updates

Commit:
no changes

JasonWHowell committed:
Update TOC.yml

JasonWHowell committed:
Update TOC.yml

BryanLa committed:
updates

rojasja committed:
Correcting misspelled using word

Commit:
Merge pull request #24888 from msebolt/freshness-review

no changes

Commit:
Added link to user voice

PRMerger10 committed:
Merge pull request #24890 from piotrci/master

Added a link to user voice to the GBL landing page.

Commit:
task 444: reverting 01_OMS-workspace.png to original graphic

Commit:
task 444: reverting to clearer non-cropped image

PRMerger16 committed:
Merge pull request #24861 from hning86/patch-11

Update how-to-create-dsvm-hdi.md

shruja committed:
update

shruja committed:
update

Commit:
Fixed parameter

PRMerger16 committed:
Merge pull request #24892 from raymondlaghaeian/patch-7

Fixed parameter

AnoopVasudavan committed:
fixing error

AnoopVasudavan committed:
fixing date

v-vikum2 committed:
updated

Commit:
Merge pull request #24894 from AnoopVasudavan/master

Master

ggailey777 committed:
Corrected link

ggailey777 committed:
Another link fix

ggailey777 committed:
Min-bar acrolinx

billmath committed:
updating

billmath committed:
updating

PRMerger16 committed:
Merge pull request #24897 from billmath/aadch1

updating

Commit:
Merge pull request #24721 from PanosPeriorellis/master

Changes to pronunciation

rojasja committed:
Very small edits correcting levels 2.3 .4 .5

rojasja committed:
file link correction on migration samples

Commit:
iris update with prereq

PRMerger10 committed:
Merge pull request #24901 from hning86/iris

iris update with prereq

Lisaco88 committed:
Merge pull request #24623 from dkkapur/master

apps and svcs as arm resources

PRMerger10 committed:
Merge pull request #24900 from billmath/dsacl1

updating

Commit:
minor edits

Lisaco88 committed:
Merge pull request #24884 from amitsriva/master

Master

sethmanheim committed:
Fix code snippets

Commit:
Acrolynx edits

Lisaco88 committed:
Merge pull request #24733 from ccompy/management-doc-tweak

Management doc tweak

GitHubber17 committed:
Merge pull request #24876 from JasonWHowell/patch-1

Update TOC.yml

GitHubber17 committed:
Merge pull request #24877 from JasonWHowell/patch-2

Update TOC.yml

GitHubber17 committed:
Merge pull request #24844 from dksimpson/dean-444

Dean 444

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

tdykstra committed:
theano feedback

Lisaco88 committed:
Merge pull request #24693 from ggailey777/ignite

Ignite

Lisaco88 committed:
Merge pull request #24797 from ggailey777/scripts

Fixed conflict in customer PR to public

MicrosoftGuyJFlo committed:
Update to session timeout per PG

bwren committed:
alert actions

sethmanheim committed:
Fix include wording

bwren committed:
payloads

tdykstra committed:
fix h3 --> h2

rmca14 committed:
Merge pull request #24867 from btardif/master

updating screenshots for ip restrictions

rmca14 committed:
Merge pull request #24656 from BryanLa/bryanla-integrating-apps

Freshness updates - integrating apps

tdykstra committed:
fix template name

PRMerger17 committed:
Merge pull request #24176 from JSeb225/patch-9

Update stream-analytics-run-a-job.md

cherylmc committed:
add Telia

tfitzmac committed:
added supported regions

Ja-Dunn committed:
Merge pull request #24855 from v-stbee/how-to-deploy-ts-guide

Adding troubleshooting guide

cherylmc committed:
update Telia

Ja-Dunn committed:
Merge pull request #24629 from v-vikum2/turborater

USER STORY 1091602 SaaS App Tutorial: TurboRater

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into providers2

Ja-Dunn committed:
Merge pull request #24634 from shruja/schoox

USER STORY 1091636: SaaS App Tutorial: Schoox

Ja-Dunn committed:
Merge pull request #24716 from shruja/agiloft

USER STORY 1093032: SaaS App Tutorial: Agiloft

Ja-Dunn committed:
Merge pull request #24841 from jimdial/patch-118

Grouped HowTo articles

ShawnJackson committed:
Update api-management-role-based-access-control.md

tdykstra committed:
fix link

ShawnJackson committed:
Update api-management-using-with-internal-vnet.md

rjagiewich committed:
Merge pull request #24724 from rojasja/master

Master

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into providers2

Commit:
spelling and grammar fixes

Lisaco88 committed:
Merge pull request #24707 from danieldobalian/master

Updated CA doc for Graph 403 Error

olivertowers committed:
Update data metadata

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPublicRepo

Commit:
updated author

PRMerger17 committed:
Merge pull request #24925 from sethmanheim/incl-fix

Fix include wording

PRMerger18 committed:
Merge pull request #24939 from cherylmc/providers2

Telia

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into providers

PRMerger12 committed:
Merge pull request #24792 from olivertowers/patch-3

Update the quickstart guide to reflect sample project changes.

PRMerger16 committed:
Merge pull request #24919 from MicrosoftGuyJFlo/PasswordFAQUpdate

Update to session timeout per PG

CarlRabeler committed:
TOC

PRMerger12 committed:
Merge pull request #24905 from sethmanheim/code-fix

Fix code snippets

PRMerger14 committed:
Merge pull request #24944 from Jim-Parker/100617updates

spelling and grammar fixes

Commit:
update author attribute

v-alje committed:
Merge pull request #24914 from MicrosoftDocs/FromPublicRepo

From public repo

tfitzmac committed:
added resource id example

ShawnJackson committed:
Merge pull request #24238 from LoriMerrick/api-management-two-articles

edit pass: api-management-two-articles

PRMerger10 committed:
Merge pull request #24953 from Jim-Parker/master

update author attribute

Commit:
Merge pull request #24038 from raiye/patch-39

Update cloud-services-guestos-update-matrix.md

PRMerger17 committed:
Merge pull request #24155 from raiye/patch-40

Update cloud-services-guestos-msrc-releases.md

rmca14 committed:
Merge pull request #24744 from tdykstra/df

Durable Functions

bwren committed:
api link

PRMerger14 committed:
Merge pull request #24954 from tfitzmac/1006resource

added resource id example

Lisaco88 committed:
Merge pull request #24951 from CarlRabeler/toc2

TOC

PRMerger20 committed:
Merge pull request #24937 from cherylmc/providers

add Telia

PRMerger15 committed:
Merge pull request #24938 from tfitzmac/1006eg

added supported regions

ShawnJackson committed:
Update active-directory-licensing-group-problem-resolution-azure-portal.md

bwren committed:
link

Commit:
Merge pull request #24904 from msebolt/freshness-review-node-node-schedule-jobs

minor edits

ShawnJackson committed:
Merge pull request #24882 from LoriMerrick/active-directory-licensing-group-problem-resolution-azure-portal3

Edit pass: active-directory-licensing-group-problem-resolution-azure-portal3

tynevi committed:
Merge pull request #24927 from bwren/la-alertactions

Log Analytics alert actions
